registration-processor/core-processor/registration-processor-demo-dedupe-stage/src/main/java/io/mosip/registration/processor/stages/demodedupe/DemodedupeProcessor.java

```diff
@@ -160,8 +160,9 @@ public MessageDTO process(MessageDTO object, String stageName) {
 				.getRegistrationStatus(registrationId, object.getReg_type(), object.getIteration());
 
 		try {
-			// Persist Demographic packet Data if packet Registration type is NEW
-			if (registrationStatusDto.getRegistrationType().equals(RegistrationType.NEW.name())) {
+			// Persist Demographic packet Data if packet Registration type is NEW or CORRECTION
+			if (registrationStatusDto.getRegistrationType().equals(RegistrationType.NEW.name())
+					||registrationStatusDto.getRegistrationType().equals(RegistrationType.CORRECTION.name())) {
```

Comment:
please remove this line -
registrationStatusDto.getRegistrationType().equals(RegistrationType.CORRECTION.name()

The correction is a generic process. It can be anything defined by country which we dont know. So we cant add hardcoded correction process here.